tools/meta/board.py

```diff
@@ -16,11 +16,21 @@ class Board:
     i2c: Optional[str] = None
     partition: int = 0
     blk: Optional[str] = None
+    baud_rate: Optional[int] = None
 
 
 # Keep this list in alphabetical order by board name
 # TODO: convert to Dictionary
 BOARDS: List[Board] = [
+    Board(
+        name="rock3b",
+        arch=SystemDescription.Arch.AARCH64,
+        paddr_top=0xecc9c000,
+        serial="serial@fe660000",
+        timer="rktimer@fe5f0000",
+        ethernet="ethernet@fe2a0000",
+        baud_rate=1500000,
```

Comment:
Assuming this is the serial baud rate, is there any reason it should be this value? If not, probably best to keep it at 115200 since that's what most other boards we operate use.

Reply:
There's nothing wrong setting the baud rate to 115200 it would just mean we would lose all the default U-Boot and debug logs that happen before the serial driver starts so it's not worth it.

@JDuchniewicz can you please add a comment saying the default UART baud rate for this platform is different from our default sDDF baud rate.